File: Lesson 79/Homework/homework.py
def count_area(*args):
    if len(args) == 1:
        radius = args[0]
        return 3.14 * radius ** 2
    elif len(args) == 2:
        length, width = args
        return length * width
    elif len(args) == 3:
        a, b, c = args
        s = (a + b + c) / 2
        return (s * (s - a) * (s - b) * (s - c)) ** 0.5

File: Lesson 79/Homework/homework.js
function countArea(...args) {
    if (args.length === 1) {
        let radius = args[0];
        return 3.14 * radius ** 2;
    } else if (args.length === 2) {
        let [length, width] = args;
        return length * width;
    } else if (args.length === 3) {
        let [a, b, c] = args;
        let s = (a + b + c) / 2;
        return Math.sqrt(s * (s - a) * (s - b) * (s - c));
    }
}
